public/gameInfo.php

```diff
@@ -87,6 +87,8 @@
             $gate = true;
         }
     }
+} elseif (str_contains($gameTitle, '[Theme - Mature]')) {
+    $gate = !($userDetails && BitSet($userDetails['websitePrefs'], $matureContentPref));
```

Comment:
Consider improving readability by distributing the not:
```
$gate = !$userDetails || !BitSet(...);
```

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1663/commits/c2190473a6afc0e5fc6ccacd1d7a6f3a524f9b9d.